plugins/mcreator-localization/help/default/structure/ground_detection.md

```diff
@@ -1,7 +1,7 @@
 This parameter controls how the structure ground will be detected.
 
-Example: If we make a structure for a water-based biome, the **first motion blocking block** 
-will generate the structure below the water as the water is not considered motion blocking block
+Example: If we make a structure for a water-based biome, the **first motion-blocking block** 
+will generate the structure below the water, as the water is not considered motion blocking block
 as the entities can go through it.
 
-The **first block** detection type will generate the structure above the water.
\ No newline at end of file
+The **first block** detection type will generate the structure above the water.

```

Comment:
EOL remove

Reply:
What does EOL stand for?

Reply:
End of line

Reply:
![image](https://github.com/user-attachments/assets/b6512e69-c51d-45a5-a797-1a9f0e51fe60)

New lines were added to the end of some of files which is unrelated change